target/linux/rtl838x/files-5.4/drivers/net/dsa/rtl838x_sw.c

```diff
@@ -30,34 +31,46 @@ struct rtl838x_mib_desc {
 	const char *name;
 };
 
-inline void rtl838x_mask_port_reg(u64 clear, u64 set, int reg)
+static inline void rtl838x_mask_port_reg(u64 clear, u64 set, int reg)
```

Comment:
This function (and many other functions below) gets referenced by a function pointer and therefore can't get inlined. Please remove the `inline` keyword.